scripts/install.bash

```diff
@@ -0,0 +1,123 @@
+AWS_VER_REGEXP_2='aws-cli\/2.\d*.\d*'
```

Comment:
Just to confirm my understanding, are we checking in the script so that we can update the pipeline to have it be packed into the orb in a subsequent PR?

Reply:
@lokst We could remove it and do this later but that was my thinking. I'm attempting to write my scripts separately in anticipation of the orb pack command.